src/gamma_inc.jl

```diff
@@ -0,0 +1,580 @@
+using Base.Math: @horner
+using Base.MPFR: ROUNDING_MODE
+#useful constants
+const acc0 = [5.0e-15 , 5.0e-7 , 5.0e-4] #accuracy options
+const big1 = [25.0 , 14.0, 10.0]
+const e0 = [0.25e-3 , 0.25e-1 , 0.14]
+const x0=[31.0 , 17.0 , 9.7]
+const alog10 = log(10)
+const rt2pin = 1.0/sqrt(2*pi)
+const rtpi = sqrt(pi) 
+const exparg = -745.1
+
+wk = zeros(30)
```

Comment:
this should be moved inside the function (sharing a common work array won't be safe for threading)